test/images/resource-consumer/README.md

```diff
@@ -0,0 +1,74 @@
+# Resource Consumer container for testing
+
+Docker image of Resource Consumer can be found in Google Container Registry as gcr.io/google_containers/resource_consumer:beta
```

Comment:
This paragraf shouldn't be the first paragraph in the doc. There should be a sub-section docker image.